sale_stock_available_info_popup/static/src/xml/qty_at_date.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2020 Tecnativa - Ernesto Tejeda
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<templates>
+    <div t-extend="sale_stock.qtyAtDate">
+        <t
+            t-jquery="t[t-if^='widget.data.virtual_available_at_date']"
+            t-operation="attributes"
+        >
+            <attribute
+                name="t-if"
+            >widget.data.immediately_usable_qty_today &lt; widget.data.qty_to_deliver and !widget.data.is_mto</attribute>
+        </t>
+    </div>
+    <div t-extend="sale_stock.QtyDetailPopOver">
+        <t t-jquery="t[t-if*='!data.is_mto']" t-operation="append">
```

Comment:
The field doesn't show up in the lines popup. It'd say you need to refine this selectors.

Reply:
It seems that v14 is also incorrect.

Reply:
The module isn't migrated to v14 or is it? :thinking:

Reply:
Sorry, it is not migrated to v14. Changes done (it´s already solved).